packages/experimental/src/experimental-list/collapsible-list/index.tsx

```diff
@@ -86,10 +118,14 @@ export const ExperimentalCollapsibleList: React.FC< CollapsibleListProps > = ( {
 	show = 0,
 	onCollapse,
 	onExpand,
+	mountOnEnter = true,
+	unmountOnExit = false,
 	...listProps
 } ): JSX.Element => {
 	const [ isCollapsed, setCollapsed ] = useState( collapsed );
-	const [ containerHeight, setContainerHeight ] = useState( 0 );
+	const [ isTransitionCollapsed, setTransitionCollapsed ] = useState(
```

Comment:
Nitpicky, but the "transition" hasn't collapsed.  Is this after the transition is complete?  `isTransitionComplete`?  Or maybe only after the transition has exited?

Reply:
This would trigger a collapse/expand transition, the name is kinda confusing, it is basically identical to `isCollapsed`, but only used to handle the `Transition` component.
I will try to come up with a better name, or maybe just `isTransitionComponentCollapsed`?

Reply:
Updated the name to the above here: https://github.com/woocommerce/woocommerce-admin/pull/7263/commits/5b42c476920bdee69d25d031bc8568b9ff6b700a
I could update it again if you had something else in mind.

Reply:
Thanks for the explanation!  I think that works.  We can always update if we think of something in the future.